### ISSUE TITLE
Unreal II [Unreal II: The Awakening] [4947003C]

### ISSUE THREAD

**Issue by hotzenplotz3000** [Cxbx-Reloaded/game-compatibility#427]
[Unreal II: The Awakening](https://en.wikipedia.org/wiki/Unreal_II:_The_Awakening)

### Description

Game is booting but has issues with intro videos.
It loops the sound but the videos won't play.
Intros can be skipped by pressing start, then it reaches the main menu.
When launching a new game, it switches to u2_retail.xbe where it starts loading but eventually crashes. Sometimes it take 5 minutes before it crash

```
---------------------------
Cxbx-Reloaded
---------------------------
Received Exception Code 0xC0000005 @ EIP := 0x000DBDF2(=XFONT_OpenBitmapFontFromMemory+0x50b95)

  Press "OK" to terminate emulation.
  Press "Cancel" to debug.
---------------------------
OK   Abbrechen   
---------------------------
```

### Dumps 

[default_xbe.txt](https://github.com/Cxbx-Reloaded/game-compatibility/files/1433973/default_xbe.txt)
[u2_retail_xbe.txt](https://github.com/Cxbx-Reloaded/game-compatibility/files/1433976/u2_retail_xbe.txt)
[KrnlDebug_default.txt](https://github.com/Cxbx-Reloaded/game-compatibility/files/1433978/KrnlDebug_default.txt)
[KrnlDebug_u2_retail.txt](https://github.com/Cxbx-Reloaded/game-compatibility/files/1433979/KrnlDebug_u2_retail.txt)

### Status
**Previous behavior:** 
None reported

**Current behavior:**
intros won't play, crash after starting new game

### Screenshots 
![2017-11-01 12_22_42-cxbx-reloaded 262810b8 oct 30 2017 _ emulating unreal ii - fps_ 59 75 ms_f_ 1](https://user-images.githubusercontent.com/30478365/32273404-a99cbe1e-bf02-11e7-856d-2a9a7f0f109f.png)
![2017-11-01 12_23_01-cxbx-reloaded 262810b8 oct 30 2017 _ emulating unreal ii - fps_ 59 49 ms_f_ 1](https://user-images.githubusercontent.com/30478365/32273408-ad3d7fa4-bf02-11e7-9b8e-abb733f7023e.png)
![2017-11-01 12_23_29-cxbx-reloaded 262810b8 oct 30 2017 _ emulating unreal ii - fps_ 4 75 ms_f_ 66](https://user-images.githubusercontent.com/30478365/32273410-aeabdf0c-bf02-11e7-8c67-98575660dccc.png)

### Additional Information

- German version
- tested build 262810b8

**Comment by jeltaqq** [comment]
### Description

Shows menus, gameplay loading functioning, but when goto ingame to dirty disc error.

### Screenshots 
![error](https://user-images.githubusercontent.com/25340049/35133844-e31e0f3e-fd15-11e7-930e-36de953d58eb.jpg)


### Additional Information

Tested with 0398d19e

**Comment by T02my** [comment]
state-partially-working

https://www.youtube.com/watch?v=nm_bIX7pamQ

BUT the AI is broken, sometimes it can't interact with you and just stand and can't be killed, sometimes it can be killed, sometimes it even shot upon you. But you can't progress further in second level Mining complex, because AI should spawn.

**Comment by T02my** [comment]
Everything is ok with https://ci.appveyor.com/project/SoullessSentinel/cxbx-reloaded/build/be58bb983f0335e407d9b0a3949ea48c3b8c7eca/job/0p7sty5ob03c50x3/artifacts

Prove https://youtu.be/foBQYIbxJaU


In other versions you have to enable "Skip rdtsc patching".

**Comment by ObiKKa** [comment]
The AI seems to be done in this video from literalmente{game}. The enemies react to the player's presence and shoot at him.

- [Unreal II: The Awakening - Mining Complex](https://m.youtube.com/watch?v=bviNT7P9tig&feature=youtu.be) (Literalmente{game}.)

**Comment by T02my** [comment]
Yes, it was fixed with @jackchentwkh's commit. I think the fix is now im main branch. Right now I'm playing on "Hell" level.

**Comment by RadWolfie** [comment]
It's not in v0.1 branch. Pull request is still open. See here for further progress: https://github.com/Cxbx-Reloaded/Cxbx-Reloaded/pull/1310

**Comment by T02my** [comment]
The game is crashing at the middle of cutscene in level "NA-KOJA ABAD -> EXCAVATION", when the alien race should appear.

To explain when it does crash, open up the video from PC walkthrought (the versions are almost the same) - https://www.youtube.com/watch?v=NiKkisjmXcw and watch it from 6:19. The game crashes on Xbox on the moment from 6:42

The save is in attachment, put it into C:\Users\*USERNAME*\AppData\Roaming\Cxbx-Reloaded\EmuDisk\Partition1\UDATA
[KrnlDebug.txt](https://github.com/Cxbx-Reloaded/game-compatibility/files/2153130/KrnlDebug.txt)

[4947003c.zip](https://github.com/Cxbx-Reloaded/game-compatibility/files/2153129/4947003c.zip)
